uwblueprint/community-fridge-kw
# feat: add new nav dropdown for admin view

## Brief description. What is this change? 

### [Update Admin navigation to include a dropdown for checkins vs scheduled donations](https://www.notion.so/uwblueprintexecs/Update-Admin-navigation-to-include-a-dropdown-for-checkins-vs-scheduled-donations-dfe6ca463e5d4efeb69d42e76155f343)



## Implementation description. How did you make this change?
* Add dropdown UI in navigation header w/ custom style
* Change `/admin/view-donations` to`/view-donations`
* Create a new route called `/admin/check-ins`


## Steps to test
1. Sign in using admin account (`name@uwblueprint.org` email)
2. Test Navigation bar
3. Check Figma designs


https://user-images.githubusercontent.com/69697180/158032753-c4aa8185-c1ec-4c8e-a340-b05cedabc667.mov
<img width="559" alt="image" src="https://user-images.githubusercontent.com/69697180/158032771-8653c2ec-ac61-4cd3-a12d-0ff8960b1e78.png">


## Checklist
